services/eventdb/main/main.go

```diff
@@ -51,6 +51,9 @@ func main() {
 	}
 	defer closer.Close()
 
+	// Report runtime and process metrics.
+	instrument.StartReportingExtendedMetrics(scope, time.Second, instrument.DetailedExtendedMetrics)
```

Comment:
You can configure this from the config file by adding
```
extended: moderate
```

Keep in mind that emitting detailed GC metrics will stop the world and cause GC pauses, so I'd advise using moderate if possible.

Reply:
dope